src/icon-manifest/languages.ts

```diff
@@ -182,6 +182,7 @@ export const languages: ILanguageCollection = {
   swig: { ids: 'swig', defaultExtension: 'swig' },
   systemverilog: { ids: 'systemverilog', defaultExtension: 'sv' },
   t4: { ids: 't4', defaultExtension: 'tt' },
+  templatetoolkit: { ids: 'tt', defaultExtension: 'tt' },
```

Comment:
In order to avoid overriding the `t4` file extension, change the `defaultExtension` for `tt` to `tt3`.